tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -194,12 +207,29 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+
+            if (cfg.cr){
+                for (var i = 0; i < domains.length; i++){
+                    if (targetSrc.indexOf(domains[i]) > 0){
+                        domainRetryIndex = i;
+                        break;
+                    }
+                }
+            }
+
             const _handleError = (evt?: any) => {
-                loadFailed = true;
                 appInsights.queue = []; // Clear the queue
                 if (!handled) {
-                    handled = true;
-                    _reportFailure(targetSrc);
+                    // start retry
+                    if (domainRetryIndex >= 0 && domainRetryCount < domains.length){
```

Comment:
Nice handling of when they don't use one of our supported domains (I almost missed that), maybe add a comment here so that someone in the future doesn't wonder (too much) why we are checking whether `domainRetryIndex` < 0.